edgebuilder-install.sh

```diff
@@ -205,28 +228,19 @@ install_node()
   DIST_NAME=$(get_dist_name "$DIST")
   DIST_NUM=$(get_dist_num "$DIST")
   DIST_TYPE=$(get_dist_type "$DIST")
-
-  if [ "$ARCH" = "x86_64" ];then
-    wget -q -O - "https://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/amd64/3004/SALTSTACK-GPG-KEY.pub" | sudo apt-key add -
-    if grep -q "deb http://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/amd64/3004 $DIST_NAME main" /etc/apt/sources.list.d/eb-saltstack.list ;then
-      echo "INFO: Salt repo already added"
-    else
-      echo "deb [arch=amd64] http://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/amd64/3004 $DIST_NAME main" | sudo tee -a /etc/apt/sources.list.d/eb-saltstack.list
-    fi
-
-  elif [ "$ARCH" = "aarch64" ];then
-    wget -q -O - "https://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/arm64/3004/SALTSTACK-GPG-KEY.pub" | sudo apt-key add -
-    if grep -q "deb http://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/arm64/3004 $DIST_NAME main" /etc/apt/sources.list.d/eb-saltstack.list ;then
-      echo "INFO: Salt repo already added"
-    else
-      echo "deb http://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/arm64/3004 $DIST_NAME main" | sudo tee -a /etc/apt/sources.list.d/eb-saltstack.list
-    fi
-  elif [ "$ARCH" = "armv7l" ];then
-    wget -q -O - "https://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/armhf/3004/SALTSTACK-GPG-KEY.pub" | sudo apt-key add -
-    if grep -q "deb http://repo.saltstack.com/py3/$DIST_TYPE/$DIST_NUM/armhf/3004 $DIST_NAME main" /etc/apt/sources.list.d/eb-saltstack.list ;then
-      echo "INFO: Salt repo already added"
+  DIST_ARCH=$(get_dist_arch "$ARCH")
+  echo "Checking dist..."
+  echo $DIST_NAME
+  if [ "$DIST_NAME" = "jammy" ]; then
+    apt-get install salt-minion
```

Comment:
Has this been done because you didn't need to install the keys for the salt repo? Just wondering because I think we should at least make sure we're installing v 3004 to be consistent with the below

Reply:
Thats because salt v3004 is available via its apt repo in ubuntu22.04

Reply:
Yes I understand that, but I think we should be using a fixed version

Reply:
oh okay, I misunderstood your previous comment, yeah, we should use a fixed version. Thanks